doc/api/errors.md

```diff
@@ -794,6 +794,12 @@ An invalid symlink type was passed to the [`fs.symlink()`][] or
 
 An attempt was made to add more headers after the headers had already been sent.
 
+<a id="ERR_HTTP_INCOMING_SOCKET_ENCODING"></a>
+### ERR_HTTP_INCOMING_SOCKET_ENCODING
+
+An attempt was made to manipulate the encoding of an incoming request packet.
+This is forbidden by [RFC7230 Section 3](https://tools.ietf.org/html/rfc7230#section-3)
```

Comment:
Nit: this is longer than 80 characters and the linter is probably going to complain about it.

Reply:
It did - I wasn't sure what to do because it was a URL and it appears to be the only one in the docs that is written this way. Still thought it was worth referencing the RFC because this is such a specific issue with a specific fix with no other options - so I left it
This looks better 👍

Reply:
I believe what we generally do is put all link URLs at the end of the file, and then reference the label here. The same could be done here.